active/Resources/StyleVersion2.md

```diff
@@ -0,0 +1,89 @@
+ThemeVersion API
+=
+
+# Background
+
+XAML (WinUI) has a resources feature where you can put objects in a "resource dictionary", then
+use the resources by referencing the key name. All of the styling for XAML controls
+is defined in a resource dictionary and referenced by the controls.
+But an application can override these by defining a resource dictionary with one or more matching keys.
+
+XAML ships such an overrides dictionary named
+[XamlControlsResources](http://msdn.microsoft.com/library/Microsoft.UI.Xaml.Controls.XamlControlsResources).

```

Comment:
Broken link. This one seems to work: [XamlControlsResources](https://docs.microsoft.com/en-us/windows/winui/api/microsoft.ui.xaml.controls.xamlcontrolsresources?view=winui-2.5).

(Same broken link appears later in the document as well.)